src/admin.py

```diff
@@ -166,11 +170,21 @@ def debug_uptime(_id, endpoint=None):
 def resave():
     # when index mappings are changed
     logger = logging.getLogger("resave")
-    for smartapi in SmartAPI.get_all(1000):
+    for smartapi in SmartAPIEntity.get_all(1000):
         logger.info(smartapi._id)
         smartapi.save()
 
 
+def refresh_metakg():
+    meta_kg = MetaKG()
+    meta_kg.construct_MetaKG()
+    docs = meta_kg.get_associations()
+    for doc in docs:
+        meta_kg_doc = MetaKGDoc(**doc)
+        meta_kg_doc._raw = decoder.compress(json.dumps(doc).encode())
```

Comment:
@sengineer0 I actually don't think we need to store a `_raw` field for metakg association docs. In `SmartAPIEntity` case, we did some transformation (for better ES indexing) so we want to store a blob of the raw doc in `_raw` field to get the exact raw data.

Reply:
oh, ok. Let's me update to remove `_raw` field in MetaKGDoc
@newgene

Reply:
It's updated. @newgene